config/custom-environment-variables.json

```diff
@@ -42,11 +44,26 @@
     },
     "allowAppointee": {
       "enabled": "ALLOW_APPOINTEE_ENABLED"
+    },
+    "allowSaveAndReturn": {
+      "enabled": "ALLOW_SAVE_RETURN"
```

Comment:
we should configure this env var as we have this one for example: POSTCODE_CHECKER_ENABLED
and then enable it in AAT and in AKS so that we can actually test it as part of this PR